types/parse/parse-tests.ts

```diff
@@ -1,124 +1,129 @@
-class GameScore extends Parse.Object {
+import { parse } from 'path';
```

Comment:
why is it here? didn't find any usage

Reply:
Oh it's accident. Sometimes "auto import" feature is annoying. Thanks you. 🙏